README.md

```diff
@@ -204,66 +206,112 @@ const toast = document.createElement('std-toast');
 console.log(toast.open); // false
 ```
 
+There will additionally be an `action` property,
+which returns or allows setting an element that provides the toast's action.
+In detail:
+
+- The getter returns the first `button` or `a` element descendant with the `slot="action"` attribute set.
+  If no such element exists,
+  it returns null.
+- The setter accepts a `button` or `a` element,
+  appends it to the toast element,
+  and sets the `slot="action"` attribute on it.
+  If any existing such elements exist,
+  the first one (i.e. the one that would be returned by the getter) is replaced.
+  If the provided value is not a `HTMLButtonElement` or `HTMLAnchorElement`,
+  a `TypeError` is thrown.
+
+To see examples of the `action` getter and setter in use,
+read on to the next section.
+
 #### Contents
 
-The `<std-toast>` element can be used as a generic container,
-but will work best (in terms of default styling and events) if the developer conforms to the following content model:
+At a technical level, the `<std-toast>` element can be used as a generic container.
+However, the authoring conformance requirements,
+as well as the API,
+steer the developer in to the following content model:
 
-- The first child node,
-  typically either a text node or a container like `<p>`,
-  provides a message.
-- If more than one child node is present,
-  then the last child element,
-  which can be either an `<a>` or a `<button>`,
-  provides a call to action.
+- All children except an action need to be [phrasing content](https://html.spec.whatwg.org/#phrasing-content-2)
+  that is not [interactive content](https://html.spec.whatwg.org/#interactive-content-2).
+  These children provide the toast's message.
+- There may be zero or one actions,
```

Comment:
Do we have any accessibility links or arguments that we can add near this section to back up the zero / one choice?

Reply:
I don't think we have any accessibility arguments for this at this point in time. Although you can imagine some (e.g. if there were a keyboard shortcut), for now I think it's more just about simplicity and "what is a toast"